test/e2e/functional/dag-noroot-branch-failed.yaml

```diff
@@ -0,0 +1,48 @@
+apiVersion: argoproj.io/v1alpha1
```

Comment:
Please move this file under test/e2e/expectedfailures since it is expected to fail

Reply:
OK